YapDatabase/Extensions/View/YapDatabaseViewTransaction.m

```diff
@@ -2980,18 +2980,19 @@ - (void)enumerateRowidsInGroup:(NSString *)group
 	
 	for (YapDatabaseViewPageMetadata *pageMetadata in pagesMetadataForGroup)
 	{
-		YapDatabaseViewPage *page = [self pageForPageKey:pageMetadata->pageKey];
-		
-		[page enumerateRowidsUsingBlock:^(int64_t rowid, NSUInteger idx, BOOL *innerStop) {
-			
-			block(rowid, pageOffset+idx, &stop);
-			
-			if (stop || [self->parentConnection->mutatedGroups containsObject:group]) *innerStop = YES;
-		}];
-		
-		if (stop || [parentConnection->mutatedGroups containsObject:group]) break;
-		
-		pageOffset += pageMetadata->count;
+        @autoreleasepool {
```

Comment:
I believe yap mostly uses tabs for indentation, which is why this diff looks goofy.

You mentioned that, for readability, you didn't bother *re-indenting* the places where we'd previously gotten this wrong, which is reasonable, but this chunk is a new file, could you keep the tabs here please? Again, just trying to minimize the upstream diff if we ever need to pull in a patch.

Reply:
I guess it's all going to change anyway since we're introducing an autoreleasepool. 🤷‍♂ TIOLI